### PR TITLE
Enable Tor DNSPort

### PR DESCRIPTION
**Pull request by BokilaLin**
Enable Tor's DNSPort support

ex: dig @container_ip -p 9453 example.com

### CODE REVIEW

**Comment by BokilaLin** [comment]
@PeterDaveHello 
I just change to use default DNS resolver port 53/udp.

I don't know somehow why i didn't need to make some capability configuration.

because it just worked, so I bother not around it right now .

File: docker-compose.yml [hunk 1]
```diff
@@ -6,5 +6,6 @@ services:
     container_name: tor-socks-proxy
     image: peterdavehello/tor-socks-proxy:latest
     ports:
+      - "127.0.0.1:53:53/udp"
```
Comment (PeterDaveHello) [inline docker-compose.yml:9]: Do you mind to temporarily remove `/udp` for consistency, and send another PR to specify tcp and udp in both `docker-compose.yml` and `Dockerfile`? Many thanks!

File: Dockerfile [hunk 2]
```diff
@@ -15,6 +15,6 @@ COPY torrc /etc/tor/
 HEALTHCHECK --timeout=10s --start-period=60s \
     CMD curl --fail --socks5-hostname localhost:9150 -I -L 'https://www.facebookcorewwwi.onion/' || exit 1
 
-EXPOSE 53 9150
+EXPOSE 53/udp 9150
```
Comment (PeterDaveHello) [inline Dockerfile:18]: Sorry that if I didn't explain it clearly enough, as 9150 port is using tcp only, I wonder if we can have a pull request to directly specify the udp/tcp part, to ensure the consistency in the configs, thanks

Comment (BokilaLin) [reply Dockerfile:18]: @PeterDaveHello 

Do you mean in Dockerfile
```
EXPOSE 53/udp 9150/tcp
```
and in docker-compose.yml
```
ports:
  -  127.0.0.1:53:53/udp
  -  127.0.0.1:9150:9150/udp
```

Right?

Comment (PeterDaveHello) [reply Dockerfile:18]: Yes, I mean after this PR merging ;)

Comment (PeterDaveHello) [reply Dockerfile:18]: (before the merging, I prefer to keep it without udp/tcp info to keep the consistency)

**Comment by PeterDaveHello** [comment]
BTW @BokilaLin, it'll be great if you'd like to also update README.md about related changes ;)
